pkg/codegen/internal/test/testdata/provider-config-schema/dotnet/Pulumi.Configstation.csproj

```diff
@@ -40,6 +40,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Pulumi" Version="3.12" />
```

Comment:
Now ideally we'd compile against the version in the repo, not published version 3.12. Is this the case?

Reply:
That'd be ProjectReference.. but perhaps this is substituted dynamically?

Reply:
These files are generated from a Pulumi schema: `schema.json`. I don't think that we support `ProjectReference` (check out `codegen/dotnet/gen.go:1986`).

Reply:
https://github.com/pulumi/pulumi/issues/7957